lib/web_ui/lib/src/engine/text/ruler.dart

```diff
@@ -24,22 +24,22 @@ String buildCssFontString({
   if (fontStyle != null) {
     result.write(fontStyle == ui.FontStyle.normal ? 'normal' : 'italic');
   } else {
-    result.write(FlutterViewEmbedder.defaultFontStyle);
+    result.write(StyleManager.defaultFontStyle);
   }
```

Comment:
This is probably a left-over from the null-safety migration, but...

Can these if-elses be default values of the parameters of the `buildCssFontString` method?

So instead of having a `required ui.FontStyle?` (why make something required AND nullable?) do:

```dart
String buildCssFontString({
  required String fontFamily,
  ui.FontStyle fontStyle = StyleManager.defaultFontStyle,
  ui.FontWeight fontWeight = StyleManager.defaultFontWeight,
  double fontSize = StyleManager.defaultFontSize,
})
```

Reply:
Nice suggestion! But I couldn't do exactly what you said because the parameter is `ui.FontStyle` but the default value is a `String` (for a good reason, I think). So I had to tweak it a bit, and it looks a lot better than it used to. See: https://github.com/flutter/engine/pull/47489/commits/c8c358dcf62e10a1148c25370fa90bf1fceecffc

Reply:
Ahhh very cool, I totally didn't notice they were different types! :P